tensorflow/compiler/mlir/lite/quantization/quantization_config.cc

```diff
@@ -60,7 +60,7 @@ bool ParseInputNodeQuantSpecs(absl::string_view node_names,
   std::vector<double> node_maxs;
   if (!max_values.empty()) {
     std::vector<std::string> node_maxs_str = absl::StrSplit(max_values, ',');
-    for (int i = 0; i < node_maxs_str.size(); i++) {
+    for (size_t i = 0; i < node_maxs_str.size(); i++) {
```

Comment:
This is inconsistent, the previous file you used static_cast<int>() for the bound and now you're changing the induction variable type (I much prefer staying with `int`)

Reply:
My apologies, my heuristic behind each individual change was to make the smallest change that doesn't modify intended behavior, whilst resolving the associated `[-Wsign-compare]` warning. 

* In some cases I use `static_cast` to cast from the type with the smaller range to the type with the larger range so that overflow cannot occur on comparison in an unintended way. 

* In other cases, specifically where the index ( sometimes an `int` ) is being compared to a `size_t` in order to check for a stop iteration condition: I just change the `int` to `size_t` ( given the iteration variable/index is not being compared to anything within the loop ). 

What are your thoughts?

Reply:
Also, I will do some cleanup, it was my intention only to submit the final commit for PR.

Reply:
I checked the usual style in MLIR, the most common way to write such loop would be either:
```
for (int i : llvm::seq<int>(node_maxs_str.size())
```
or
```
for (int i = 0, e = node_maxs_str.size();  i < e; i++)
```

(both of them also convey the intent that the size won't change during the loop)

In this particular case here though, this seems like an obvious for-range loop: 

```
for (const std::string& node_max : node_maxs_str) {
```

Reply:
Ok, thank you very much for the feedback!